### PR TITLE
Esa 72 - Add internationalization

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[main]
+host = https://www.transifex.com
+
+[VAS.VAS]
```
Comment (vguzmanp) [inline .tx/config]: We will need to change this as soon as the transifex project is created.

**Comment by aameijide** [comment]
Checked with Longomatch, the build/m4 has more files here. Maybe some files must not been pushed

**Comment by vguzmanp** [comment]
Ready to review again

**Comment by aameijide** [comment]
👍

**Comment by ylatuya** [comment]
Missing autogen.sh script

File: po/LINGUAS [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# please keep this list sorted alphabetically
+#
```
Comment (ylatuya) [inline po/LINGUAS]: Include the same languages as in longomatch

Comment (vguzmanp) [inline po/LINGUAS]: Including the languages but not the translated .po files makes `make check` to fail. Is this what we want?

Comment (ylatuya) [inline po/LINGUAS]: We should include the translations too, copied from longomatch and cleaned up. There should be a script to automate it for when we merge other libraries from longomatch

Comment (vguzmanp) [inline po/LINGUAS]: Fixed using `intltool-update`

**Pull request closed without merging**